### PR TITLE
Ensure Inverter/Charger details are also fetched for non-vebus services 

### PR DESCRIPTION
**Pull request by blammit**

### CODE REVIEW

**Review by DanielMcInnes — COMMENTED** [review]
WHen I open & shut the control cards in a wasm build, I see the following in the browser console output:
QString::arg: Argument missing: Inverter / Charger, MultiPlus 48/3000/35-50
The cerbo is running 'boat/motorhome demo 1'
It could be just because of the demo mode

File: components/InverterModeDialogLauncher.qml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+/*
+** Copyright (C) 2023 Victron Energy B.V.
```
Comment (DanielMcInnes) [inline components/InverterModeDialogLauncher.qml:2]: I can't remember whether we need the right year (2024) or not. Here and elsewhere.

**Review by DanielMcInnes — COMMENTED** [review]
Would you mind fixing this unrelated bug as well?

qrc:/qt/qml/Victron/VenusOS/pages/vebusdevice/PageVeBusError11Menu.qml:27: ReferenceError: valid is not defined

I think it should be 'isValid'

File: components/InverterModeDialogLauncher.qml (hunk 1 above)
Comment (blammit) [reply components/InverterModeDialogLauncher.qml]: Sounds good, done

**Comment by blammit** [comment]
> Would you mind fixing this unrelated bug as well?
> 
> qrc:/qt/qml/Victron/VenusOS/pages/vebusdevice/PageVeBusError11Menu.qml:27: ReferenceError: valid is not defined
> 
> I think it should be 'isValid'

Done, and fixed others also.

**Comment by blammit** [comment]
> WHen I open & shut the control cards in a wasm build, I see the following in the browser console output: QString::arg: Argument missing: Inverter / Charger, MultiPlus 48/3000/35-50 The cerbo is running 'boat/motorhome demo 1' It could be just because of the demo mode

There's a build issue where the translations don't get generated properly after the first build. If I run the build again then the translation arguments get filled in correctly, do you get that too?

**Comment by DanielMcInnes** [comment]
> > WHen I open & shut the control cards in a wasm build, I see the following in the browser console output: QString::arg: Argument missing: Inverter / Charger, MultiPlus 48/3000/35-50 The cerbo is running 'boat/motorhome demo 1' It could be just because of the demo mode
> 
> There's a build issue where the translations don't get generated properly after the first build. If I run the build again then the translation arguments get filled in correctly, does that fix the issue for you?

Yes it does, all good

**Review by DanielMcInnes — APPROVED** [review]
lgtm

**Comment by blammit** [comment]
@DanielMcInnes I've added [cee5c49](https://github.com/victronenergy/gui-v2/pull/710/commits/cee5c49a61d38c153f7e16cc684d37e9eb70aed8) and [eb9dabf](https://github.com/victronenergy/gui-v2/pull/710/commits/eb9dabfd0590b66f1ceddc079da20b0860d19ddf) as Matthijs asked to have the list shown when clicking on the overview widget, if there are multiple devices. Could you please check this? There are multiple devices in mock mode.

**Comment by DanielMcInnes** [comment]
> @DanielMcInnes I've added [cee5c49](https://github.com/victronenergy/gui-v2/pull/710/commits/cee5c49a61d38c153f7e16cc684d37e9eb70aed8) and [eb9dabf](https://github.com/victronenergy/gui-v2/pull/710/commits/eb9dabfd0590b66f1ceddc079da20b0860d19ddf) as Matthijs asked to have the list shown when clicking on the overview widget, if there are multiple devices. Could you please check this? There are multiple devices in mock mode (although the inverters don't have proper states there yet).

I tested this against a real installation with multiple inverter / chargers, looks good

File: components/InverterModeDialogLauncher.qml (hunk 1 above)
Comment (DanielMcInnes) [reply components/InverterModeDialogLauncher.qml]: Still a few 2023 copyright notices

**Review by DanielMcInnes — APPROVED** [review]
still a few 2023 copyright notices, otherwise lgtm

File: components/InverterModeDialogLauncher.qml (hunk 1 above)
Comment (blammit) [reply components/InverterModeDialogLauncher.qml]: Oops, fixed, thanks

**Pull request merged**